### ISSUE TITLE
MCO doesn't wipe out added `command` in spec

### ISSUE THREAD

**Issue by jlebon** [openshift/machine-config-operator#93]
```
oc edit ds machine-config-server
# add to spec e.g.:
command: ["/bin/bash"]
args: ["-c", "echo hello"]
```

The operator correctly restores the `args` var, but not the `command`, presumably because there is no `command` defined in the manifest either.

**Comment by vrutkovs** [comment]
No longer happens in 4.2 (2019-07-09-052923) but still affects 4.1 (4.1.4)

**Comment by runcom** [comment]
this is changing with #739 but looks like something for kube upstream to correctly restore it (with #739 we'll have a command in manifest as well)

**Comment by vrutkovs** [comment]
Fixed in 4.2 by https://github.com/openshift/machine-config-operator/pull/850 (`command` is explicitly specified in manifests).

4.1 fix: https://github.com/openshift/machine-config-operator/pull/940
